src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyView.js

```diff
@@ -22,6 +25,24 @@ const Body = SelectAuthenticatorVerifyViewBody.extend({
       }
     }];
   },
+
+  // If the `deviceKnown` attribute is true, we should put the signed_nonce method to the top of authenticator list.
+  _sortMethodOptionsIfNeeded(methodOptions) {
+    // Check if the `deviceKnown` attribute is true
+    const deviceKnown = this.options.currentViewState.relatesTo?.value?.deviceKnown;
+
+    if (deviceKnown) {
+      const signedNonceIndex = methodOptions.findIndex((e) => e.value === 'signed_nonce');
+
+      if (signedNonceIndex >= 0) {
```

Comment:
if index is 0, then we don't really need to do anything right? Since it is already ordered
This should be `if (signedNonceIndex > 0) {`

Reply:
Good catch. I updated the code